components/actionbar/index.css

```diff
@@ -12,42 +12,37 @@
  */
 
 .spectrum-ActionBar {
-	--spectrum-actionbar-popover-background-color: var(--spectrum-gray-25);
-	--spectrum-actionbar-popover-border-color: var(--spectrum-gray-400);
-
+	/* Layout */
 	--spectrum-actionbar-height: var(--spectrum-action-bar-height);
-	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-100);
+	--spectrum-actionbar-minimum-width: var(--spectrum-action-bar-minimum-width);
+	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-medium-size-extra-large);
+	--spectrum-actionbar-spacing-label-to-actiongroup: var(--spectrum-action-bar-label-to-action-group-area);
+	--spectrum-actionbar-spacing-action-group-edge: var(--spectrum-action-bar-edge-to-content-area);
+	--spectrum-actionbar-close-button-to-counter: var(--spectrum-action-bar-close-button-to-counter);
 
 	/* item counter field label */
 	--spectrum-actionbar-item-counter-font-size: var(--spectrum-font-size-100);
 	--spectrum-actionbar-item-counter-line-height: var(--spectrum-line-height-100);
 	--spectrum-actionbar-item-counter-color: var(--spectrum-neutral-content-color-default);
 
 	/* emphasized variation colors */
-	--spectrum-actionbar-emphasized-background-color: var(--spectrum-informative-background-color-default);
-	--spectrum-actionbar-emphasized-item-counter-color: var(--spectrum-white);
+	--spectrum-actionbar-emphasized-background-color: var(--spectrum-neutral-content-color-default);
+	--spectrum-actionbar-emphasized-item-counter-color: var(--spectrum-gray-25);
```

Comment:
Could some of these for emphasized by handled by changes to existing custom properties, instead of creating new ones and repeating some declarations later in the CSS?
```
.spectrum-ActionBar--emphasized {
    --spectrum-actionbar-item-counter-color: var(--spectrum-gray-25);
    --spectrum-actionbar-popover-background-color: var(--spectrum-neutral-content-color-default);
    ...
}
```